vars/piperPipelineStageInit.groovy

```diff
@@ -84,6 +84,10 @@ import static com.sap.piper.Prerequisites.checkScript
      * @possibleValues `true`, `false`
      */
     'skipCheckout',
+    /**
+    * Optional if you skip the checkout. Then you need to unstash your workspace to get the e.g. configuration.
```

Comment:
I think this might be more clear? Since it's optional normally, but mandatory when you skip the checkout
```suggestion
    * Mandatory if you skip the checkout. Then you need to unstash your workspace to get the e.g. configuration.
```